packages/foreman/foreman/foreman.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Foreman
+Documentation=https://theforeman.org
+After=network.target remote-fs.target nss-lookup.target
+
+[Service]
+Type=simple
+User=foreman
+TimeoutSec=300
+WorkingDirectory=/usr/share/foreman
+ExecStart=/usr/bin/scl enable tfm "rails server -e $FOREMAN_ENV -p $FOREMAN_PORT"
+Environment=FOREMAN_ENV=production FOREMAN_PORT=3000
```

Comment:
By default we are but have supported a standalone service for a long time. With switching it to systemd we've dropped some more technical debt. It would be interesting to see if we can use Puma and set up Apache as a reverse proxy to use websockets, but that wasn't the direct goal of this change.